src/scripts.js

```diff
@@ -60,3 +68,22 @@ function displayHydrationInfo(user, date) {
     <p>${pastWeekHydrationObjects[5].date}: ${pastWeekHydrationObjects[5].numOunces} ounces</p>
     <p>${pastWeekHydrationObjects[6].date}: ${pastWeekHydrationObjects[6].numOunces} ounces</p>`;
 }
+
+function displaySleepInfo(user, date) {
+  const pastWeekSleepObjects = sleepRepository.getSleepDataByWeek(5, date);
+  sleepSection.innerHTML = `
+    <h2>SLEEP!</h2>
+    <p>Hours slept last night: ${sleepRepository.getSleepHoursByDate(5, "2019/09/22")}</p>
```

Comment:
So parameters with the getSleepHoursByDate don't use the user and date arguments passed in?